package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
Surprises me too. Perhaps has something to do with the way tape/babel-register work? IDK